db/models/model.js

```diff
@@ -7,4 +7,11 @@ const selectAllTopics = () => {
     })
 };
 
-module.exports = { selectAllTopics };
\ No newline at end of file
+const selectArticleById = (article_id) => {
+    const queryString = `SELECT * FROM articles WHERE article_id=${article_id};`;
```

Comment:
⚠️  You are in danger of sql injection here. We need to use the `$1` syntax and pass an array of value(s) to prevent this ⚠️  Have a look at the node-postgres notes section  parameterized query examples for a refresher.